tools/sdk/include/user_interface.h

```diff
@@ -575,9 +575,16 @@ enum wps_cb_status {
     WPS_CB_ST_UNK,
 };
 
+#ifdef NO_EXTRA_4K_HEAP
```

Comment:
Please add a descriptive comment about what the #define means, or at least reference the issue.

@igrr In this case, should the change be moved into a patch that gets applied over the original user_interface.h file? I'm thinking of migration to future sdk versions, and what will happen with our own changes to this file.